### ISSUE TITLE
NTR: Modifier: Medium vessel involvement of vasculitis (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9314]
New term request Modifier: Medium vessel involvement of vasculitis (ang_130514223917):   UMLS: 4510989

**Comment by pnrobinson** [comment]
@MickeySegal
is this being refered to?
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2908435/
I suspect it would be better to make this a term rather than a modifier

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000658
- label: Medium vessel vasculitis
- parents: [Vasculitis by anatomical site (HP:0034784)](https://hpo.jax.org/app/browse/term/HP:0034784)
- pmid: PMID:19946711
- def: A type of vasculitis (inflammation of blood vessel walls) that affects medium-sized blood vessels. By common usage, vasculitis predominantly refers to arterial disease. Medium-sized vasculitis is observed in diseases including polyarteritis nodosa, Kawasaki disease, and cutaneous polyarteritis nodosa.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9314
- synonyms: Medium-size-vessel vasculitis(NONE)
